Index: src/test/java/controller/init.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>user login u ali p 123\r\nduel new second-player username rounds 1\r\nnext phase\r\nselect hand 3\r\nsummon\r\nselect hand 2\r\nset\r\nnext phase\r\nnext phase\r\nnext phase\r\nnext phase\r\nnext phase\r\nnext phase\r\nselect hand 4\r\nsummon\r\nnext phase\r\nnext phase\r\nnext phase\r\nnext phase\r\nnext phase\r\nnext phase\r\nselect hand 3\r\nset\r\nnext phase\r\nnext phase\r\nnext phase\r\nnext phase\r\nnext phase\r\nnext phase\r\nselect hand 2\r\nsummon\r\nnext phase\r\nnext phase\r\nnext phase\r\nnext phase\r\nnext phase\r\nnext phase\r\nselect hand 2\r\nsummon\r\nnext phase\r\nnext phase\r\nnext phase\r\nnext phase\r\nnext phase\r\nnext phase\r\nselect hand 3\r\nsummon\r\nnext phase\r\nnext phase\r\nnext phase\r\nnext phase\r\nnext phase\r\nnext phase\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/controller/init.txt b/src/test/java/controller/init.txt
--- a/src/test/java/controller/init.txt	(revision 9092344a371e3276347a86490767d1303ada03b7)
+++ b/src/test/java/controller/init.txt	(date 1623842239453)
@@ -44,6 +44,7 @@
 next phase
 next phase
 select hand 3
+css
 summon
 next phase
 next phase
Index: src/test/java/controller/TributeTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/controller/TributeTest.java b/src/test/java/controller/TributeTest.java
new file mode 100644
--- /dev/null	(date 1623842925675)
+++ b/src/test/java/controller/TributeTest.java	(date 1623842925675)
@@ -0,0 +1,13 @@
+package controller;
+
+import org.junit.jupiter.api.BeforeEach;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TributeTest {
+    @BeforeEach
+    void init(){
+
+    }
+
+}
\ No newline at end of file
Index: src/test/java/controller/tricky.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/controller/tricky.txt b/src/test/java/controller/tricky.txt
new file mode 100644
--- /dev/null	(date 1623843338665)
+++ b/src/test/java/controller/tricky.txt	(date 1623843338665)
@@ -0,0 +1,6 @@
+user login u ali p 123
+duel new second-player username rounds 1
+gc The Tricky
+select hand 5
+active effect
+1
\ No newline at end of file
